node/blockAPI/shardBlock.go

```diff
@@ -53,7 +54,20 @@ func (sbp *shardAPIBlockProcessor) GetBlockByHash(hash []byte, withTxs bool) (*a
 		return nil, err
 	}
 
-	return sbp.convertShardBlockBytesToAPIBlock(hash, blockBytes, withTxs)
+	blockAPI, err := sbp.convertShardBlockBytesToAPIBlock(hash, blockBytes, withTxs)
```

Comment:
duplicated code with the metaBlock.go L58. The only difference is how the storer unit is passed. Please extract in a new function.

Reply:
refactored this part